bootstrap/src/main/java/sun/security/ssl/SSLEngineImpl.java

```diff
@@ -479,6 +482,22 @@ private void initHandshaker() {
         handshaker.setEnableSessionCreation(enableSessionCreation);
     }
 
+
```

Comment:
This is not complete implementation as in JDK, but prevents stacktraces and still allows to use previous implementation.